### ISSUE TITLE
请问训练完报这个错误是什么意思

### ISSUE THREAD

**Issue by yunchangxiaoguan** [PaddlePaddle/PaddleSeg#908]
python3 train.py --config configs/my.yaml --save_interval 500 --save_dir output/my
错误如下：

021-03-21 11:22:40 [INFO]	[TRAIN] epoch=140, iter=980/1000, loss=2.8907, lr=0.000309, batch_cost=1.9775, reader_cost=1.51363, ips=2.0227 samples/sec | ETA 00:00:39
2021-03-21 11:23:00 [INFO]	[TRAIN] epoch=142, iter=990/1000, loss=3.3142, lr=0.000173, batch_cost=2.0295, reader_cost=1.56442, ips=1.9709 samples/sec | ETA 00:00:20
2021-03-21 11:23:20 [INFO]	[TRAIN] epoch=143, iter=1000/1000, loss=2.7183, lr=0.000020, batch_cost=1.9681, reader_cost=1.50462, ips=2.0324 samples/sec | ETA 00:00:00
<class 'paddle.nn.layer.conv.Conv2D'>'s flops has been counted
Customize Function has been appied to <class 'paddle.nn.layer.norm.SyncBatchNorm'>
Cannot find suitable count function for <class 'paddle.nn.layer.pooling.MaxPool2D'>. Treat it as zero Macs.
<class 'paddle.nn.layer.pooling.AdaptiveAvgPool2D'>'s flops has been counted
<class 'paddle.nn.layer.pooling.AvgPool2D'>'s flops has been counted
Cannot find suitable count function for <class 'paddle.nn.layer.activation.Sigmoid'>. Treat it as zero Macs.
<class 'paddle.nn.layer.common.Dropout'>'s flops has been counted
Total Flops: 32887304192     Total Params: 2528485

**Comment by wuyefeilin** [comment]
你好，这个不是报错哈，这是在统计模型的flops和params时打印出来的信息

**Comment by yunchangxiaoguan** [comment]
> 你好，这个不是报错哈，这是在统计模型的flops和params时打印出来的信息

好的，谢谢

**Comment by yunchangxiaoguan** [comment]
> 你好，这个不是报错哈，这是在统计模型的flops和params时打印出来的信息

好的，谢谢

**Comment by timelovery** [comment]
我想问一下我这是什么原因：

2024-12-23 10:54:35 [INFO]      [TRAIN] epoch: 9, iter: 280/300, loss: 0.0921, lr: 0.001402, batch_cost: 0.5546, reader_cost: 0.21817, ips: 7.2130 samples/sec, max_mem_reserved: 20447 MB, max_mem_allocated: 15204 MB | ETA 00:00:11
2024-12-23 10:54:41 [INFO]      [TRAIN] epoch: 10, iter: 290/300, loss: 0.1018, lr: 0.001452, batch_cost: 0.5878, reader_cost: 0.25591, ips: 6.8051 samples/sec, max_mem_reserved: 20447 MB, max_mem_allocated: 15204 MB | ETA 00:00:05
2024-12-23 10:54:46 [INFO]      [TRAIN] epoch: 10, iter: 300/300, loss: 0.0960, lr: 0.001502, batch_cost: 0.5676, reader_cost: 0.22530, ips: 7.0474 samples/sec, max_mem_reserved: 20447 MB, max_mem_allocated: 15204 MB | ETA 00:00:00
<class 'paddle.nn.layer.conv.Conv2D'>'s flops has been counted
Cannot find suitable count function for <class 'paddle.nn.layer.norm.LayerNorm'>. Treat it as zero FLOPs.
<class 'paddle.nn.layer.common.Linear'>'s flops has been counted
<class 'paddle.nn.layer.common.Dropout'>'s flops has been counted
Cannot find suitable count function for <class 'paddleseg.models.backbones.transformer_utils.Identity'>. Treat it as zero FLOPs.
Cannot find suitable count function for <class 'paddle.nn.layer.activation.GELU'>. Treat it as zero FLOPs.
Cannot find suitable count function for <class 'paddleseg.models.backbones.transformer_utils.DropPath'>. Treat it as zero FLOPs.
Cannot find suitable count function for <class 'paddle.nn.layer.common.Dropout2D'>. Treat it as zero FLOPs.
<class 'paddle.nn.layer.norm.BatchNorm2D'>'s flops has been counted
<class 'paddle.nn.layer.activation.ReLU'>'s flops has been counted
Total Flops: 36384432216     Total Params: 3715170。

导入输出的模型进行预测时，又显示每一层都没有weight和bias：
2024-12-23 11:04:58 [INFO]      Loading pretrained model from /mnt/Member_dir/huangyinghua/projects/code/PpSeg/output/DXtest1/iter_300/model.pdparams
2024-12-23 11:04:58 [WARNING]   encode.double_conv.0._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.0._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.0._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.0._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.0._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.0._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.1._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.1._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.1._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.1._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.1._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.double_conv.1._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.1._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.1._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.1._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.1._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.1._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.1._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.2._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.2._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.2._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.2._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.2._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.0.2._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.1._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.1._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.1._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.1._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.1._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.1._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.2._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.2._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.2._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.2._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.2._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.1.2._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.1._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.1._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.1._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.1._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.1._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.1._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.2._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.2._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.2._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.2._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.2._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.2.2._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.1._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.1._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.1._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.1._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.1._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.1._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.2._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.2._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.2._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.2._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.2._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   encode.down_sample_list.3.2._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.0._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.0._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.0._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.0._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.0._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.0._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.1._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.1._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.1._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.1._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.1._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.0.double_conv.1._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.0._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.0._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.0._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.0._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.0._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.0._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.1._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.1._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.1._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.1._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.1._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.1.double_conv.1._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.0._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.0._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.0._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.0._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.0._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.0._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.1._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.1._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.1._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.1._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.1._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.2.double_conv.1._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.0._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.0._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.0._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.0._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.0._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.0._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.1._conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.1._conv.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.1._batch_norm.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.1._batch_norm.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.1._batch_norm._mean is not in pretrained model
2024-12-23 11:04:58 [WARNING]   decode.up_sample_list.3.double_conv.1._batch_norm._variance is not in pretrained model
2024-12-23 11:04:58 [WARNING]   cls.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   cls.bias is not in pretrained model
2024-12-23 11:04:58 [WARNING]   conv.weight is not in pretrained model
2024-12-23 11:04:58 [WARNING]   conv.bias is not in pretrained model
2024-12-23 11:04:58 [INFO]      There are 0/112 variables loaded into UNet.
2024-12-23 11:04:58 [INFO]      Start to predict...
